website/content/docs/api-gateway/upgrade-specific-versions.mdx

```diff
@@ -21,13 +21,15 @@ Ensure that the following requirements are met prior to upgrading:
 - You should have the ability to run `kubectl` CLI commands.
 - `kubectl` should be configured to point to the cluster containing the installation you are upgrading.
 - You should have the following permission rights on your Kubernetes cluster:
-  * `HTTPRoute.read`
-  * `TCPRoute.read`
-  * `ReferencePolicy.create`
+  - `HTTPRoute.read`
+  - `TCPRoute.read`
+  - `ReferencePolicy.create`
 - (Optional) The [jq](https://stedolan.github.io/jq/download/) command line processor for JSON can be installed, which will ease route retrieval during the upgrade process.
 
 ## Procedure
 
+-> **NOTE** When you see `VERSION` in examples of commands or configuration settings, replace `VERSION` with the version number of the release you are installing, like `0.2.0`. If there is a lower case "v" in front of `VERSION` the version number needs to follow the "v" as is `v0.2.0`
+
```

Comment:
@Jeff-Apple I think we should add a shell-session that sets the version for the doc. This will help the reader when issuing the commands the need the version later on.  

```
export VERSION=0.2.0
```